test/hotspot/jtreg/compiler/ciReplay/TestInlining.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2016, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I think the copyright date should just be 2021, right?

Reply:
I copied TestVMNoCompLevel.java and then modified it, so I think keeping the start year is correct, as long as that content was non-trivial.  But I can change it if you feel strongly about it.

Reply:
Okay, makes sense then to keep it as is.